list-targets/action.yaml

```diff
@@ -39,15 +39,26 @@ runs:
       env:
         HEAD_SHA: ${{github.event.pull_request.head.sha}}
 
+    - run: go install github.com/hashicorp/terraform-config-inspect@latest
+      shell: bash
     - uses: suzuki-shunsuke/tfaction/list-working-dirs@main
       id: list-working-directory-configs
+    - uses: suzuki-shunsuke/tfaction/get-global-config@main
+      id: global-config
+    - uses: suzuki-shunsuke/tfaction/list-module-callers@main
+      id: list-module-callers
+      if: steps.global-config.outputs.disable_update_local_path_module_caller == 'false'
```

Comment:
It seems the feature is enabled by default, but as I said I want to disable this feature by default.

https://github.com/suzuki-shunsuke/tfaction/pull/1528#issuecomment-1941061896

And I think the output name should be `update_local_path_module_caller` because the feature is disabled by default.

Reply:
Rnamed in https://github.com/suzuki-shunsuke/tfaction/pull/1528/commits/f6d9d8bbd92de64c11cba3989aa727abbb6be62b